litmus-portal/frontend/public/locales/en/translation.yaml

```
@@ -1034,9 +1034,16 @@ createWorkflow:
       showDetails: Show Details
       showProp: Show Properties
       addProbe: Please add probes to see the data
-      addNewProbe: "+ Add a new Probe"
+      addNewProbe: '+ Add a new Probe'
       back: Back
       finish: Finish
+    env:
+      envHeading: Tune the environment variables of the experiment
+      showMoreEnv: Show more environment variables
+      showLessEnv: Show less environment variables
+      key: Key Turnables
```

It should be like this i think
```suggestion
      key: Key Tunables
```